Dockerfile

```diff
@@ -48,72 +35,66 @@ RUN apt-get update && \
     libnss3-tools \
     iproute2 \
     software-properties-common \
-    nano && \
-#    gpg-agent \
-#    python-setuptools \
-#    gcc && \
-# Node setup
-# Node
-  curl -sL https://deb.nodesource.com/setup_${NODEJS_VERSION} | sudo -E bash - && \
+    nano
+
+# Add the ARGs here so that changing them doesn't invalidate the previous layers' cache
+ARG FIREFOX_STABLE_VERSION=127.0.2
+ARG LIGHTHOUSE_VERSION=12.1.0
+ARG NODEJS_VERSION=20
+
+# Install Node.js with nvm
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash && \
+  export NVM_DIR="$HOME/.nvm" && \
+  [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && \
+  nvm install $NODEJS_VERSION
+
 # Install browsers
-# Chrome
-  wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
-  echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
-# Chrome stable has to be manually downloaded and installed (only latest stable version is available via apt-get)
-  wget --no-verbose -O /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_STABLE_VERSION}_amd64.deb && \
-    apt install -y /tmp/chrome.deb && \
-    rm /tmp/chrome.deb && \
+# Note: Google have started removing old Chrome versions, so we cannot specify a version here. We
```

Comment:
Where did you find this info?

We need to be able to specify a known version so that we know what versions customers are running